app/services/slack/webhook.py

```
@@ -0,0 +1,256 @@
+            {
+                "type": "header",
+                "text": {
+                    "type": "plain_text",
+                    "text": f"🔴 Breeze Buddy Alert - {evaluator_name}",
+                    "emoji": True,
+                },
+            },
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"*Score:*\n0.0 (FAILURE)"},
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)